addons/godot-firebase/firestore/firestore.gd

```diff
@@ -40,27 +53,49 @@ func collection(path : String) -> FirestoreCollection:
     else:
         return collections[path]
 
+
+func query(query : FirestoreQuery):
+    request = REQUESTS.QUERY
+    var body : Dictionary = {
+        structuredQuery = query.query,
+       }
+    var url : String = _base_url + _extended_url + _query_suffix
+    _request_list_node.request(url, ["Authorization: Bearer " + auth.idtoken], true, HTTPClient.METHOD_POST, JSON.print(body))
```

Comment:
This should use the FirestoreCollection.authorization_header or whatever that constant is.

Reply:
FirestoreCollection._authorization_header is just a string containing "Authorization: Bearer". Should I use that instead?